lua/harpoon/ui.lua

```diff
@@ -65,8 +64,16 @@ M.toggle_quick_menu = function()
         contents[idx] = string.format("%d %s", idx, file)
     end
 
+    vim.api.nvim_buf_set_name(bufh, "harpoon-menu")
```

Comment:
Oh, I think this is really the one I was missing!

Reply:
Yes